readthedocs/api/v2/views/model_views.py

```diff
@@ -130,7 +130,7 @@ def active_versions(self, request, **kwargs):
             Project.objects.api(request.user),
             pk=kwargs['pk'],
         )
-        versions = project.versions.filter(active=True)
+        versions = project.versions(manager='internal').filter(active=True)
```

Comment:
Is this the normal way to handle this? Is it not normally `project.versions.internal.filter`?

Reply:
yes this is the normal way to do this if we use multiple managers.
ref: [Dajngo Docs](https://docs.djangoproject.com/en/1.11/topics/db/queries/#using-a-custom-reverse-manager)